aspnetcore/fundamentals/websockets.md

```diff
@@ -30,7 +30,7 @@ This article explains how to get started with WebSockets in ASP.NET Core. [WebSo
 
   * Windows 8 / Windows Server 2012 or later
   * IIS 8 / IIS 8 Express
-  * WebSockets must be enabled in IIS (See the [IIS/IIS Express support](#iisiis-express-support) section.)
+  * WebSockets must be enabled (See the [IIS/IIS Express support](#iisiis-express-support) section.).
```

Comment:
@anurse if you and @tdykstra  approve of my suggested changes, I can make them for you.

The best rule of thumb for punctuating in and around parenthetical remarks is that the sentence should be valid if you remove the parentheses and everything inside them.
@tdykstra 	 please review. I'm guessing it should be 
WebSockets must be enabled (See the [IIS/IIS Express support](#iisiis-express-support) section~.~).

Reply:
I would be more inclined to go with WebSockets must be enabled. (See the IIS/IIS Express support section.) or omit the parens.